src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes, useKeyOnly } from '../../utils/propUtils'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ *
+ * @see Breadcrumb
+ */
+function BreadcrumbSection(props) {
+  const {
+    active, children, className, link, href, onClick, ...rest,
+  } = props
+
+  /**
+   * Handles onClick event. If onClick is not defined, does nothing.
+   * */
```

Comment:
Reserve doc block comments (`/**`) for docs and public methods/members.  Since this function cannot be used outside of this class, the doc block can be removed.